cmd/manager/main.go

```diff
@@ -228,6 +264,9 @@ func main() {
 			KlusterletWorkLister:     listersworkv1.NewManifestWorkLister(klusterletWorksInformer.GetIndexer()),
 			HostedWorkInformer:       hostedWorksInformer,
 			HostedWorkLister:         listersworkv1.NewManifestWorkLister(hostedWorksInformer.GetIndexer()),
+			KlusterletConfigInformer: klusterletconfigInformer,
+			KlusterletConfigLister:   listerklusterletconfigv1alpha1.NewKlusterletConfigLister(klusterletWorksInformer.GetIndexer()),
```

Comment:
curious, why we need to set an indexer, why can't we get lister from informer?

Reply:
Good point, it's just a style copy from `KlusterletWorkLister` and  `HostedWorkLister`, now look back it seems like a redudent.

Removed.